### DISCUSSION TITLE
Show mod customizability

### DISCUSSION THREAD

**Discussion by Rekunan** [ppy/osu#16054] [Ideas]
This is a suggestion to add a visual indicator, either in the tooltip or as a separate icon, which shows that a mod is customizable, kinda like how there's a text to show the compatibility of a mod.

**Comment by bdach** [comment]
probably something best addressed in the future mod select redesign (https://www.figma.com/file/NkdREXr6wFkA99oLw8PnBv/Client%2FMod-Selection?node-id=0%3A1)
